Rubberduck.Core/UI/Command/Annotations/AnnotateCommandBase.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NLog;
+using Rubberduck.Parsing.Annotations;
+using Rubberduck.Parsing.Rewriter;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+
+namespace Rubberduck.UI.Command.Annotations
+{
+    public abstract class AnnotateCommandBase : CommandBase
+    {
+        private readonly IRewritingManager _manager;
+        private readonly IAnnotationUpdater _updater;
+
+        protected AnnotateCommandBase(IRewritingManager manager, IAnnotationUpdater updater) 
+            : base(LogManager.GetCurrentClassLogger())
+        {
+            _manager = manager;
+            _updater = updater;
+        }
+
+        protected virtual IAnnotation GetAnnotation(Declaration target)
```

Comment:
Why the `GetSomthing` names. Wouldn't `Something` be a lot more in line with the style of the code base?